nextflow_schema.json

```diff
@@ -468,6 +468,24 @@
                     "description": "A csv file with vep_plugins as header, and then paths to vep plugin files. Paths to pLI_values.txt and LoFtool_scores.txt are required.",
                     "schema": "assets/vep_plugin_files_schema.json"
                 },
+                "filter_variants_hgnc_ids": {
+                    "type": "string",
+                    "pattern": "^\\S+\\.(csv|tsv)$",
+                    "description": "A tsv/csv file with a `#hgnc_ids` column header, and then one numerical HGNC ID per row. E.g. `4281`, not `HGNC:4281`.",
```

Comment:
In the file specific schema it looked fine to have `HGNC:4281` as well as `4281`

Also shouldn't the header just be `hgnc_ids`? Alternatively update file schema.

Reply:
Either I missed changing the description, or messed up the merge. Will fix.